.dockerignore

```diff
@@ -2,7 +2,12 @@
 .git
 .gitignore
 tests
+_site
+CONTRIBUTING.md
 Dockerfile
+Dockerfile.archive
 docker-compose.yml
-_site
 Gemfile.lock
+Jenkinsfile
+LICENSE
+README.md
```

Comment:
These may actually be needed, as they contain the copyright / licensing info (which should generally be distributed with each copy)

Reply:
Good point. I also thought about that. That's what code reviews are good for.
I've updated the commit and keep these two files in the `_site` folder.